rotkehlchen/db/dbhandler.py

```diff
@@ -382,10 +388,8 @@ def set_setting(
             write_cursor: 'DBCursor',
             name: Literal[
                 'version',
-                'last_write_ts',
-                'last_data_upload_ts',
                 'premium_should_sync',
-                'ongoing_upgrade_from_version',
```

Comment:
Maybe instead of using literals it would make sense to have an enum like we do for `class CacheType(Enum):`. You can make it serializableEnumMixing